app/views/search/search.html.erb

```diff
@@ -8,28 +8,36 @@
   <div class="column-two-thirds">
     <form id="dgu-search-box" action="/search" method="GET">
       <label for="q" class="visuallyhidden">Search</label>
-      <input id="q" name="q" type="text" class="form-control dgu-search-box__input" value="<%= @query %>"/>
-      <button type="submit" class="dgu-search-box__button">Find data</button>
+      <input id="q" name="q" type="text" class="form-control dgu-search-box__input" value="<%= @query %>"/><button type="submit" class="dgu-search-box__button">Find data</button>
     </form>
   </div>
 </div>
 
-<div class="grid-row results">
+<form class="dgu-filters">
   <div class="column-one-third">
     <%= render 'filters' %>
   </div>
+  <div class="column-two-thirds">
+    <%= render 'sort' %>
+  </div>
+</form>
```

Comment:
This is causing problems on small screens
<img width="464" alt="screen shot 2017-08-17 at 09 59 44" src="https://user-images.githubusercontent.com/69542/29404475-042c238a-8333-11e7-8b1d-1dc92ca03eba.png">